VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/video-call/video-call.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { VHConference, VHEndpoint, VHParticipant } from '../store/models/vh-conference';
+import { VHHearing } from 'src/app/shared/models/hearing.vh';
+
+@Component({
+    selector: 'app-video-call',
+    standalone: false,
+    templateUrl: './video-call.component.html',
+    styleUrls: ['../waiting-room-global-styles.scss', './video-call.component.scss']
+})
+export class VideoCallComponent {
+    @Input() isJudge: boolean;
+    @Input() isJohRoom: boolean;
+    @Input() canToggleParticipantsPanel: boolean;
+    @Input() isPrivateConsultation: boolean;
+    @Input() vhParticipant: VHParticipant;
+    @Input() hasCaseNameOverflowed: boolean;
+    @Input() vhConference: VHConference;
+    @Input() connected: boolean;
+    @Input() outgoingStream: MediaStream | URL;
+    @Input() isSupportedBrowserForNetworkHealth: boolean;
```

Comment:
I don't think this needs to be an input, move the logic into the deviceType service and call the method there